command/daemon.go

```diff
@@ -586,7 +586,7 @@ func createValueStore(ctx context.Context, cfgIndexer config.Indexer) (indexer.I
 			l.EnsureDefaults()
 		}
 		pebbleOpts.Levels[numLevels-1].FilterPolicy = nil
-		pebbleOpts.Cache = pbl.NewCache(1 << 30) // 1 GiB
+		pebbleOpts.Cache = pbl.NewCache(2 << 30) // 2 GiB
```

Comment:
Let's parameterise this as a config. As is it would change the setting for every node that  deploys the built image